editor/store/effects/posts.js

```diff
@@ -0,0 +1,320 @@
+/**
```

Comment:
For more sanity and also because it's easier to use `async/await` this way, I extracted some effects handler into their own files.